src/lib/bootstrap.php

```diff
@@ -19,6 +19,19 @@
 $loader = require __DIR__.'/../app/autoload.php';
 ZLoader::register($loader);
 
+// set current domain for using it as container parameter
+// http://symfony.com/doc/current/cookbook/configuration/external_parameters.html
+if (strpos(__DIR__, '/src/lib') !== false) {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/src/web';
+} else {
+    $_SERVER['SYMFONY__ASSETS_BASE_URL'] = '//' . $_SERVER['HTTP_HOST'] . '/web';
+}
+
```

Comment:
I just did the following:
- clone my fork
- remove this code in bootstrap.php and the line in config.yml
- execute composer install
- call http://localhost/src/install

Then I see this:
![screen_installer](https://cloud.githubusercontent.com/assets/277531/8052901/0e20627a-0e8c-11e5-9671-541dc5104494.jpg)

Which was the very first reason for this topic ;-)